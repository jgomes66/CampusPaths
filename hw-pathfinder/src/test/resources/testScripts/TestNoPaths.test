# Simple test to see what happens when there is no path
# between 2 buildings

CreateGraph simpleGraph

AddNode simpleGraph v1
AddNode simpleGraph v2
AddNode simpleGraph v3

AddEdge simpleGraph v1 v2 1

FindPath simpleGraph v1 v3

