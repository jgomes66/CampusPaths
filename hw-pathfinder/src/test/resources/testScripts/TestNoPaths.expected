# Simple test to see what happens when there is no path
# between 2 buildings

created graph simpleGraph

added node v1 to simpleGraph
added node v2 to simpleGraph
added node v3 to simpleGraph

added edge 1.000 from v1 to v2 in simpleGraph

path from v1 to v3:
no path found

